qa-covid19.planx-pla.net/portal/gitops.json

```
@@ -918,15 +941,31 @@
           },
           {
             "field": "hhinc_mean",
-            "name": "Mean Household Income"
+            "name": "Mean Household Income (US Currency"
           },
           {
             "field": "med_hhinc",
-            "name": "Median Household Income"
+            "name": "Median Household Income (US Currency)"
           },
           {
             "field": "cardiovasc_death_rate",
-            "name": "Cardiovascular Death Rate"
+            "name": "Cardiovascular Death Rate (annual number of deaths per 100,000 people)"
```

```suggestion
            "name": "Cardiovascular Death Rate (per 100,000 people)"
```
trying to shorten it since it's a table column name